src/problem-analysis/state-of-the-art/chess-engines.tex

```diff
@@ -1,4 +1,108 @@
 % textidote: ignore begin
 \subsection{Chess engines}\label{subsec:chess.engines}
+
+Chess engines are defined by \href{https://www.chess.com}{\verb{Chess.com}} as such:
 % textidote: ignore end
-Hello engines~\cite{alpha-go-zero}.
+\begin{quote}
+    A chess engine is simply a software program that plays and analyzes chess.
+    The word “engine” simply refers to a kind of high-powered program that does a
+    lot of searching and processing similar to a search engine~\cite{chess-com-chess-engines}.
+\end{quote}
+
+So the goal of a chess engine is to understand all the game mechanics and then evaluate a position.
+This evaluation is the most important thing a chess engine should be able to do, but as this is a very abstract
+problem there exists many ways to solve the problem which will be discussed below.
+
+\subsubsection{Conventional/brute force engines}\label{subsubsec:conventional-engines}
+
+A conventional chess engine can be simply described as a brute force approach to solve chess.
+This means that the main limiting factor of how good the engine will be is the computational power of the system.
+Such a chess engine consists of two parts, an evaluation function and a search function.
+Chess is a very complex game without a clear-cut solution, this is because if you wanted to brute force all the possible
+moves that the board could be in you would have to search \( 10^{111} \) different positions, that is more atoms than
+there are in the universe, and with current technology this would be impossible to compute~\cite{atoms}.
+
+To ease this computational burden, chess engines have a value called depth or search depth.
+The depth value can be understood as how deep the engine should look into the future, as going to the end is not
+possible, to give the best move at some current position.
+
+The search function can be further optimized in different ways, but what the optimizations in essence are, are
+clever ways to search for moves that are more likely to be good before others, that will allow you to cut out moves that
+are guaranteed to be worse than the already searched move.
+This can be visualized through a tree structure as seen in~%\ref{fig:chess_pos-tree}.
```

Comment:
@SebastianMygind I think compilation fails due to the `%`

Reply:
I have removed it, but it is still giving me errors